src/modules/notifications/Notifier.ts

```diff
@@ -14,24 +13,36 @@ export default class Notifier {
         sound = null,
         setting = null,
         image = null,
+        htmlLessMessage = null,
```

Comment:
Is this necessary?
Could we instead just use something like:
```js
const tempEl = document.createElement('div');
tempEl.innerHTML = message;
const strippedMessage = tempEl.innerText.replace(/  +/g, ' ');
```
Only thing that I could see we need to do is replace any `<br>` with `\n` as they already get converted to `<br/>` within the notifier.

Reply:
I also remove the pokemoney symbol, somewhere. And i think there also where a span...

Reply:
It should automatically remove them with the code above and only take the text.

Reply:
Oh, i replaced the pokemoney-symbol, with some text (see the notification for offlineprogress). Should have said that in my first comment...
I think i will keep the htmlLessMessage, only for that case, and then use your code for all other cases? Is that fine?

Reply:
It turns out that the notification just removes html (at least in firefox). So i am now replacing <br> with \n.
I kept the htmlLessMessage-code for the offline message (it had the pokedollar-symbol)